### PR TITLE
replace str.format with f-strings

### PR DESCRIPTION
**Pull request by Alex23rodriguez**
(and a little bit of black formatting)

### CODE REVIEW

**Comment by Alex23rodriguez** [comment]
Removed formatting

File: hooks/pre_gen_project.py [hunk 1]
```diff
@@ -39,20 +39,20 @@
...
           "See https://www.python.org/dev/peps/pep-0008/#package-and-module-names "
-          "for naming standards.".format(module_name))
+          "for naming standards.")
     invalid_inputs.append("module_name")
 
 if invalid_inputs:
-    print("\nYou have entered invalid configuration values for: "
-          "{}".format(", ".join(invalid_inputs)))
+    print("\nYou have entered invalid configuration values for:",
+          ", ".join(invalid_inputs))
```
Comment (klamann) [inline hooks/pre_gen_project.py:55]: I'm pretty sure this won't work, can you test it?

File: tests/version_check.py [hunk 2]
```diff
@@ -24,21 +24,20 @@
...
-            "failed" if expect_fail else "succeeded"))
+        print(f"Python {platform.python_version()} {status[expect_fail]} as expected")
     else:
-        print("Python {} should have {}, but actually {}".format(
-            platform.python_version(),
-            "failed" if expect_fail else "succeeded",
-            "failed" if actual_fail else "succeeded"))
+        print(
+            f"{platform.python_version()} should have {status[expect_fail]},",
+            f"but actually {status[actual_fail]}",
```
Comment (klamann) [inline tests/version_check.py:33]: please add "Python" before printing the version

**Review by klamann — CHANGES_REQUESTED** [review]
found two minor issues, looks good otherwise :)

File: hooks/pre_gen_project.py (hunk 1 above)
Comment (Alex23rodriguez) [reply hooks/pre_gen_project.py:55]: Tested. works as long as all the values in invalid_inputs are strings (which they must be, based on previous code)

File: tests/version_check.py (hunk 2 above)
Comment (Alex23rodriguez) [reply tests/version_check.py]: I missed that. fixed

**Comment by klamann** [comment]
thanks for contributing this :)

**Review by klamann — APPROVED** [review]

**Pull request merged**